src/Activity/TrainWithExample/index.js

```diff
@@ -21,10 +21,14 @@ class TrainWithExample extends React.Component {
   };
 
   recordExampleActivity = userAnswer => {
+    console.log(this.props.sessionRef);
     this.newActivityRef.child("activity_type").set("example");
     let image = parallelogramData[this.state.index].src;
     if (this.props.activityChosen === "mammals"){
       image = mammalsData[this.state.index].src;
+      this.newActivityRef.child("topic").set("mammals")
+    }else{
+      this.newActivityRef.child("topic").set("parallelograms");
```

Comment:
This is recording data in the database.

As one session will have a single topic,  think it is better to record the topic for the sessions and not for each activity of the session. 

You should move this code in another function that is called on creation of the session.
```
...sessionRef.child("topic")(activityChosen)
```

Also do not leave `console.log` in the code.